pages/add/form/detail-info-fields.js

```diff
@@ -26,7 +26,7 @@ export default {
   },
   related_creators: {
     type: Creators,
-    label: `Name any creators, contributors, partners. Comma separated.`,
+    label: `Name any creators, contributors, partners. Tab separated.`,
```

Comment:
Comma is no longer supported for some reason. I have filed a ticket on the library's repo.

See https://github.com/mozilla/network-pulse/pull/1364/files#diff-3f3823e4d9a300e71bad95c88168c569

Reply:
This is a list that we can specify (see https://github.com/i-like-robots/react-tags/blob/master/README.md#delimiters-optional) so instead of rewriting this code to advocate tabs, let's keep all of this the same but make sure to specify a `<ReactTags delimiters={[',']} ...>` ourselves.

Reply:
```suggestion
    label: `Name any creators, contributors, partners. comma-separated.`,
```